pelicanconf.py

```diff
@@ -29,10 +29,10 @@
 
 PLUGIN_PATHS = ['plugins']
 PLUGINS = ['headerid']
-HEADERID_LINK_CHAR = '&para;'
+HEADERID_LINK_CHAR = '&nbsp;'
```

Comment:
Neither of &para and &nbsp looks like a good symbol.  Sphinx NTD theme uses a 'link' symbol though I don't know how it is done.  One example is: https://yyc.solvcon.net .

Reply:
The symbol is changed to the [Font Awesome link icon](https://fontawesome.com/icons/link?style=solid) in this commit :)

The `&nbsp` (white space) is used to reserve element space, or CSS hover does not work on it.

Reply:
I misunderstood.  Thanks.